app/src/main/java/fr/free/nrw/commons/di/CommonsApplicationModule.java

```diff
@@ -63,6 +63,12 @@ public ContentProviderClient provideContributionContentProviderClient(Context co
         return context.getContentResolver().acquireContentProviderClient(CONTRIBUTION_AUTHORITY);
     }
 
+    @Provides
+    @Named("revertCount")
+    public SharedPreferences provideRevertCountSharedPreferences( Context context) {
+        return context.getSharedPreferences("revertCount", MODE_PRIVATE);
```

Comment:
Do we really need a separate shared pref for revert counts? It could be part of default prefs itself. Your call. :)

If you feel the need for a separate pref then I would suggest renaming it to something more generic such as `userStatistics` pref.